src/lib/tracer.ts

```diff
@@ -20,7 +20,6 @@ export function init() {
         // Noop
       },
     },
-    debug: !PRODUCTION_ENV,
```

Comment:
`PRODUCTION_ENV` is `true` for both staging and production, so this was likely never enabled.  
In this version, debug mode is disabled by default.  

Reference: https://github.com/DataDog/dd-trace-js/blob/v5.x/docs/API.md#custom-logging  

To enable debug mode, set `DD_TRACE_DEBUG=true`.